src/ucs/datastruct/strided_alloc.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2018.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_STRIDED_ALLOC_H_
+#define UCS_STRIDED_ALLOC_H_
+
+#include "queue_types.h"
+
+#include <ucs/type/status.h>
+#include <ucs/sys/compiler_def.h>
+#include <ucs/sys/math.h>
```

Comment:
minor: you include it in `alloc.c` file, seems no need to have it here

Reply:
it's for UCS_KBYTE